pkg/codegen/go/gen.go

```diff
@@ -645,7 +649,7 @@ func (pkg *pkgContext) genInputTypes(w io.Writer, t *schema.ObjectType, details
 	}
 	fmt.Fprintf(w, "}\n\n")
 
-	genInputMethods(w, name, name+"Args", name, details.ptrElement)
```

Comment:
Should the `resourceType` parameter be `true` here? I think that `genInputTypes` is called for non-resource types as well.

Reply:
Oops, I think I meant to default that arg to false. I'll double-check that.